fspiop-api/documents/Tracing v1.0.md

```diff
@@ -0,0 +1,221 @@
+---
+showToc: true
+---
+# Distributed Tracing Support for OpenAPI Interoperability
+
+## _Tracing for the OpenAPI Interoperability Specification_
+
+
+**Table of Content**
+
+1. [Preface](#1-preface)
+
+    1.1 [Conventions Used in This Document](#11-conventions-used-in-this-document)
+
+    1.2 [Document Version Information](#12-document-version-information)
+
+2. [Introduction](2-introduction)
+
+    2.1 [Open API for FSP Interoperability Specification](#21-open-api-for-fsp-interoperability-specification)
+
+3. [Tracing Specification](#3-tracing-specification)
+
+    3.1 [Tracing Data Model](#31-tracing-data-model)
+
+    3.2 [Generating a Trace](#32-generating-a-trace)
+
+    3.3 [Continuing a Trace](#33-continuing-a-trace)
+
+    3.4 [Forwarding a Trace](#34-forwarding-a-trace)
+
+4. [Examples](#4-examples)
+
+5. [References](#5-references)
+
+**Table of Tables**
+
+[Table 1 – Data model of HTTP header fields for Tracing](#table-1)
+[Table 2 – Data model for Tracing values](#table-2)
+[Table 3 – Supported version-formats](#table-3)
+[Table 4 – Data model for _tracestate_ list member values](#table-4)
+
+## 1. Preface
+
+This section contains information about how to use this document.
+
+### 1.1 Conventions Used in This Document
+
+This document uses the notational conventions for **2HEXDIGLC**, **16HEXDIGLC** and **32HEXDIGLC** defined in W3C Trace Context Recommendation <sup>[1](https://www.w3.org/TR/trace-context-1/#traceparent-header-field-values)</sup>.
+
+The following conventions are used in this document to identify the specified types of information.
+
+|Type of Information|Convention|Example|
+|---|---|---|
+|**Elements of the API, such as resources**|Boldface|**/authorization**|
+|**Variables**|Italics within curly brackets|_{ID}_|
+|**Glossary terms**|Italics on first occurrence; defined in _Glossary_|The purpose of the API is to enable interoperable financial transactions between a _Payer_ (a payer of electronic funds in a payment transaction) located in one _FSP_ (an entity that provides a digital financial service to an end user) and a _Payee_ (a recipient of electronic funds in a payment transaction) located in another FSP.|
+|**Library documents**|Italics|User information should, in general, not be used by API deployments; the security measures detailed in _API Signature_ and _API Encryption_ should be used instead.|
+
+### 1.2 Document Version Information
+
+|Version|Date|Change Description|
+|---|---|---|
+|**1.0**|2021-04-01|Initial version|
+| | | |
+
+## 2. Introduction
+
+This document details the Distributed Tracing standards to be supported for Open APIs for Interoperability (hereafter cited as the API) to ensure _tracing_ and _observability_ within the API system, and to/from API consumers.
```

Comment:
As I assume that this is valid for all the APIs within Mojaloop (e.g. FSP Interoperability API, Third Party API, ..), I would suggest the below. See issue https://github.com/mojaloop/mojaloop-specification/issues/15 for more information. 

```suggestion
This document details the Distributed Tracing standards to be supported for the Mojaloop API Suite (hereafter cited as the API) to ensure _tracing_ and _observability_ within the API system, and to/from API consumers.
```

Reply:
And another comment is that "the API" is never used after the "..hereafter cited as the API..", so the "hereafter cited as the API" can be removed.